metadata.json

```diff
@@ -29,6 +29,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">=4.4.0 <5.0.0"
+    },
+    {
+      "name": "jethrocarr/initfact",
+      "version_requirement": ">=0.0.1"
```

Comment:
Could you add an upper bound to the version number, or pin this to a specific version? Just something to keep things from breaking as you update your initfact module and make changes that I might not b keeping up with.